os/board/esp32_devkit/src/esp32_boot.c

```diff
@@ -128,6 +133,11 @@ void esp32_devKit_mount_partions(void)
 
 void esp32_board_initialize(void)
 {
+	/*Init SPI RAM*/
+#ifdef CONFIG_SPIRAM_SUPPORT
+	esp_spiram_init_cache();
+	esp_spiram_init();
+#endif
```

Comment:
esp32_board_initialize runs before os initialization.
so this function always do jobs that's not depend os service, like(malloc).
I checked functions esp_spiram_init_cache and esp_spiram_init call position in esp,
it can be called at the time that os not start actually.